Assets/MixedRealityToolkit/Providers/UnityInput/MouseDeviceManager.cs

```diff
@@ -21,10 +21,15 @@ public class MouseDeviceManager : BaseDeviceManager, IMixedRealityExtensionServi
         /// <summary>
         /// Constructor.
         /// </summary>
+        /// <param name="registrar">The <see cref="IMixedRealityServiceRegistrar"/> instance that loaded the service.</param>
         /// <param name="name">Friendly name of the service.</param>
         /// <param name="priority">Service priority. Used to determine order of instantiation.</param>
         /// <param name="profile">The service's configuration profile.</param>
-        public MouseDeviceManager(string name, uint priority, BaseMixedRealityProfile profile) : base(name, priority, profile) { }
+        public MouseDeviceManager(
+            IMixedRealityServiceRegistrar registrar, 
```

Comment:
>  [](start = 52, length = 1)

nit: a lot of these new constructors have extra spaces at the end of their lines. Please remove.